community/milind/Zarr_file_example/Zarr_file_example.py

```diff
@@ -1,25 +1,33 @@
 @fused.udf
-def udf(bbox: fused.types.Bbox = None, layer: str = "ndvi", time: int = 2, target_shape: list = [512,512]):
+def udf(bbox: fused.types.Bounds = None, layer: str = "ndvi", time: int = 2, target_shape: list = [512,512]):
```

Comment:
```suggestion
def udf(bounds: fused.types.Bounds = None, layer: str = "ndvi", time: int = 2, target_shape: list = [512,512]):
```

Using a viewport gdf now actually requires to use `bounds` as the keyword name, otherwise this will not work anymore